cmd/did-method-cli/createconfigcmd/createconfig.go

```diff
@@ -240,22 +290,24 @@ func createFlags(startCmd *cobra.Command) {
 	startCmd.Flags().StringP(outputDirectoryFlagName, "", "", outputDirectoryFlagUsage)
 }
 
-func createConfig(parameters *parameters) (map[string][]byte, error) {
+func createConfig(parameters *parameters) (map[string][]byte, map[string][]byte, error) {
```

Comment:
Is linter working for this project? This is long, complex function.

Reply:
Yeah, linter definitely worked, it complained for this I believe, until I refactored it below the cyclo limits.